managed-symbols-available/test.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "managed-symbols-available",
   "enabled": true,
+  "requiresSdk": false,
```

Comment:
I don't know if this is meaningful to run without the SDK. The symbols are part of the SDK.